src/ch/icalservice.ts

```diff
@@ -4,8 +4,9 @@ import { RacePlan } from "./models";
 import { addDays } from "date-fns";
 import { getWeekDistance, render, renderDist } from "./rendering";
 
-function toDate(d: Date): [number, number, number] {
-  return [d.getUTCFullYear(), 1 + d.getUTCMonth(), d.getUTCDate()];
+// public for testing
+export function toDate(d: Date): [number, number, number] {
+  return [d.getFullYear(), 1 + d.getMonth(), d.getDate()];
```

Comment:
This is the code change: When Local and UTC dates do not match we get an off-by-one error in the ical generation.

Reply:
Very nice, thanks for the fix!